lib/matplotlib/axes/_axes.py

```diff
@@ -188,7 +188,7 @@ def get_xlabel(self):
         label = self.xaxis.get_label()
         return label.get_text()
 
-    def set_xlabel(self, xlabel, fontdict=None, labelpad=None, **kwargs):
+    def set_xlabel(self, xlabel, fontdict=None, labelpad=None, loc=None, **kwargs):
```

Comment:
needs to be documented

Reply:
Happy to add docs if there's a chance for this PR